mbid_mapping/manage.py

```diff
@@ -199,7 +199,9 @@ def cron_build_all_mb_caches(ctx):
 @click.pass_context
 def cron_update_all_mb_caches(ctx):
     """ Update all mb entity metadata cache in ListenBrainz. """
-    ctx.invoke(update_canonical_release_data)
+
+    # In this context we want to use mb_conn, not lb_conn, like the functions that follow
+    update_canonical_release_data(False)
```

Comment:
Couldn't this be called like `ctx.invoke(update_canonical_release_data, use_lb_conn=False)` instead or does it need to be called separately from this context?

Reply:
Unsure. click is full of poorly documented black magic, that I didn't even bother trying this.

Reply:
I figured that was possibly the case, looking at https://click.palletsprojects.com/en/stable/advanced/#invoking-other-commands